Examples/OneSignalDemo/app/src/main/java/com/onesignal/sdktest/model/MainActivityViewModel.java

```diff
@@ -396,6 +397,24 @@ public void onFailure() {
                 });
             }
         });
+
+        logoutEmailButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                OneSignal.logoutEmail(new OneSignal.EmailUpdateHandler() {
+                    @Override
+                    public void onSuccess() {
+                        OneSignal.onesignalLog(OneSignal.LOG_LEVEL.DEBUG, "Logout email ended successfully");
+                        MainActivityViewModel.this.getActivity().runOnUiThread(() -> userEmailTextView.setText(""));
```

Comment:
Like the extra logging and UI update!

Reply:
No change needed in the PR but one nit is it would be nice if the example app changes where in a different commit for easier reviewing. Also smaller specific commits means easier to resolve any merge conflicts if they come up on other branches.

Reply:
Yeah, I agree. My bad. I should have been able to separate the UI change in another commit. I needed it for testing.